lib/cors-anywhere.js

```diff
@@ -215,6 +215,11 @@ function onProxyResponse(proxy, proxyReq, proxyRes, req, res) {
   delete proxyRes.headers['set-cookie2'];
 
   proxyRes.headers['x-final-url'] = requestState.location.href;
+
+  if (req.headers['mock-http-response-code']) {
+    proxyRes.statusCode = parseInt(req.headers['mock-http-response-code']);
```

Comment:
You should verify that it's a valid status code prior to assigning it (i.e. 3 digits). The current implementation will cause the server to crash if an invalid value is passed (`RangeError [ERR_HTTP_INVALID_STATUS_CODE]`).